common/list/lists-en.html

```diff
@@ -0,0 +1,400 @@
+<strong>&lt;dd&gt;</strong>...&lt;/dd&gt;
+&lt;/dl&gt;</code></pre>
+
+<h2>Unstyled lists </h2>
+<p>Unstyled list:</p>
+<ul class="list-unstyled">
+	<li>List item 1</li>
+	<li>List item 2
+	<ul>
+		<li>List item 2a</li>
+		<li>List item 2b</li>
+	</ul>
+	</li>
+	<li>List item 3</li>
+</ul>
+<p>Unstyled nested list:</p>
+<ul>
+	<li>List item 1</li>
+	<li>List item 2</lu>
+	<ul class="lst-none">
+		<li>List item 2a</li>
+		<li>List item 2b</li>
+	</ul>
+	</li>
+	<li>List item 3</li>
+</ul>
+<h3>Code</h3>
+<pre><code>//Unstyled list:
+&lt;ul <strong>class=&quot;list-unstyled&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ul&gt;
+
+//Unstyled nested list:
+&lt;ul&gt;
+	&lt;li&gt;List item 1
+		&lt;ul <strong>class=&quot;lst-none&quot;</strong>&gt;
+			&lt;li&gt;List item 1a&lt;/li&gt;
+			&lt;li&gt;List item 1b&lt;/li&gt;
+		&lt;/ul&gt;
+	&lt;/li&gt;
+&lt;/ul&gt;</code></pre>
+
+<h2> Numbered lists </h2>
+<ol>
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<ol class="lst-num">
+			<li>List item 2a</li>
+			<li>List item 2b</li>
+		</ol>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol&gt;
+&lt;li&gt;...
+	&lt;ol <strong>class=&quot;lst-num&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/li&gt;
+&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2>Alpha lists − Lower alpha </h2>
+<ol>
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<ol class="lst-lwr-alph">
+			<li>List item 2a</li>
+			<li>List item 2b</li>
+		</ol>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol&gt;
+&lt;li&gt;...
+	&lt;ol <strong>class=&quot;lst-lwr-alph&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/li&gt;
+&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2> Alpha lists − Upper alpha</h2>
+<ol class="lst-upr-alph">
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<ol>
+			<li>List item 2a</li>
+			<li>List item 2b</li>
+		</ol>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol <strong>class=&quot;lst-upr-alph&quot;</strong>&gt;
+&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2>Roman numeral − Lower Roman </h2>
+<ol>
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<ol class="lst-lwr-rmn">
+			<li>List item 2a</li>
+			<li>List item 2b</li>
+		</ol>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol&gt;
+&lt;li&gt;...
+	&lt;ol <strong>class=&quot;lst-lwr-rmn&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;/li&gt;
+&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2> Roman numeral − Upper Roman </h2>
+<ol class="lst-upr-rmn">
+	<li>List item 1</li>
+	<li>
+		List item 2
+		<ol>
+			<li>List item 2a</li>
+			<li>List item 2b</li>
+		</ol>
+	</li>
+	<li>List item 3</li>
+</ol>
+<h3>Code</h3>
+<pre><code>&lt;ol<strong> class=&quot;lst-upr-rmn&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+
+<h2> List inline </h2>
+<div class="panel panel-default">
+	<div class="panel-body">
```

Comment:
Remove the panel because the panel is out of scope to illustrate this working example
```suggestion
```